src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -33,13 +33,32 @@ export class BaseEditCourseForm extends React.Component {
     super(props);
 
     this.state = {
-      open: true,
+      open: false,
```

Comment:
We starting closed again?

Reply:
Yup. After going through usability testing with partners, we realized they don't seem to intuitively know that the course runs are collapsible components and so we want to try and make the course collapsed by default so they encounter the behavior sooner.